modal/sandbox.py

```diff
@@ -244,6 +257,24 @@ async def wait(self, raise_on_termination: bool = True):
                     raise SandboxTerminatedError()
                 break
 
+    async def tunnels(self) -> List[api_pb2.TunnelData]:
```

Comment:
Should this be synchronicity-wrapped so that it can be async without the caller needing to be async?

Reply:
Also super ignorant question about the return type: is it a tunnel or data from a tunnel?

Reply:
It's tunnel metadata. I've updated the comment to match.